### PR TITLE
minor ipm fixes

### PR DESCRIPTION
**Pull request by ryantking**
Found a few small issues when doing the gloo port of this command

### CODE REVIEW

File: docs/content/reference/helm/gloo_mesh_enterprise/latest/_index.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 
 ---
-title: "v1.1.0-beta7"
+title: "v1.0.10"
```
Comment (joekelley) [inline docs/content/reference/helm/gloo_mesh_enterprise/latest/_index.md:3]: Not a result of your code change, but i believe this is unintended. we should have a `1.0` channel for 1.0 releases and a `latest` channel for 1.1.0-beta releases. cc @ashleywang1

**Review by ashleywang1 — APPROVED** [review]

**Pull request merged**